README.md

```diff
@@ -149,6 +153,17 @@ Before starting, make sure you have the following installed:
 - [Our bluetooth Protocol Standard](https://docs.basedhardware.com/developer/Protocol/)
 - [Plugins](https://docs.basedhardware.com/developer/Plugins/)
 
+## Plug-in requirements
```

Comment:
I think the only requirement should be follow the prompt structure.

If there's something weird on a PR, of course we will not merge it, but as long as some people find that useful/fun/engaging, I don't see this requirements that strict.